reports-backend/src/main/java/org/jboss/da/reports/backend/impl/DependencyTreeGeneratorImpl.java

```diff
@@ -30,13 +30,13 @@
     private AproxConnector aproxConnector;
 
     @Inject
-    SCMConnector SCMConnector;
+    SCMConnector scmConnector;
 
     @Override
     public GAVDependencyTree getDependencyTree(SCMLocator scml) throws ScmException,
             PomAnalysisException {
-        return SCMConnector.getDependencyTreeOfRevision(scml.getScmUrl(), scml.getRevision(),
-                scml.getPomPath());
+        return scmConnector.getDependencyTreeOfRevision(scml.getScmUrl(), scml.getRevision(),
+                scml.getPomPath(), Optional.ofNullable(scml.getRepositories()));
```

Comment:
Why not use Empty list when repositories is null? What is the semantic differences between empty optional and empty list?